src/components/QandA/QandA.jsx

```diff
@@ -3,60 +3,118 @@ import QuestionList from './QuestionList.jsx';
 import './QAstyles/QandA.css';
 import server from '../../serverRequests.js';
 
-const QandA = ({prod}) => {
+function QandA ({prod}) {
   const [search, setSearch] = useState('');
-  const [count, setCount] = useState(1);
-  const [questions, setQuestions] = useState(['Question_1', 'Question_2', 'Question_3', 'Question_4'])
+  const [qCount, setQCount] = useState(2);
+  const [aCount, setACount] = useState(1);
+  const [productID, setProductID] = useState('');
+  const [answersID, setAnswersID] = useState([]);
+  const [results, setResults] = useState([]);
+  const [questionID, setQuestionID] = useState('642440');
+  const [questions, setQuestions] = useState([]);
+  const [answers, setAnswers] = useState([]);
+  const [questionHelp, setQuestionHelp] = useState(0);
+  const [reported, setReported] = useState(false);
+  const [questionAnswer, setQuestionAnswer] = useState({
+    id: '',
+    body: '',
+    date: '',
+    answerName: '',
+    helpfulness: '',
+    photos: []
+  })
 
-  //axios
+  // test (console)
   const data = () => {
-    return console.log('prod = ', prod)
+    return console.log('answers = ', answers)
   }
 
   useEffect(() => {
```

Comment:
I think you can skip this UseEffect entirely. You don't need productID as state since you're getting it as props, and you're not passing it down to other components any further.